tone-analyzer/v3.js

```diff
@@ -70,6 +71,7 @@ ToneAnalyzerV3.prototype.tone = function(params, callback) {
     return;
   }
   const contentType = params.isHTML ? 'text/html' : 'text/plain';
+  const contentLanguage = params.language ? params.language : 'en';
```

Comment:
No need to set a default here. If there is no params.language, then the header should be omitted.